integrationtest/org/daisy/dotify/formatter/test/resource-files/marker-ref/marker-ref-page-vol-input.obfl

```diff
@@ -17,15 +17,15 @@
 			<footer/>
 		</default-template>
 	</layout-master>
-	<volume-template sheets-in-volume-max="10" use-when="(= $volume 1)">
+	<volume-template sheets-in-volume-max="1000" use-when="(= $volume 1)">
```

Comment:
Why was this and subsequent similar changes needed?
It is because suddenly a volume break occurs where we didn't have a volume break in the past?

Reply:
I've made the test better by making it independent of how volume breaking works, by using forced volume breaks.